controllers/skirunsController.js

```diff
@@ -0,0 +1,39 @@
+// // controllers/albumsController.js
```

Comment:
So many commented out albums files! Was this just here for reference, or were you actually using it?